### PR TITLE
Add 'Back-to-top' button for partners list page

### PR DESCRIPTION
**Pull request by lalit97**
The partners page is quite long now. A 'back to top' button was needed.

Bug: T170113

### CODE REVIEW

**Comment by Samwalton9** [comment]
Could you add a screenshot of what this button looks like, please? :)

**Comment by lalit97** [comment]
back-to-top2

**Comment by lalit97** [comment]
Hello @Samwalton9 I thought you just missed this one, please have a look whenever you are free :)

**Comment by Samwalton9** [comment]
Hey lalit - thanks for the reminder. I'm actually going to wait on reviewing this. We've got a lot of moving pieces being merged in to the tool this month and it might just reduce complications to hold off on merging this PR until then. Sorry about that - we'll get to it in a few weeks :)

**Comment by lalit97** [comment]
Okay, no problem :)

**Comment by jsnshrmn** [comment]
I rebased this so I can check it against our current code base and review.

File: TWLight/static/css/local.css [hunk 1]
```diff
@@ -748,3 +748,28 @@ CLEARING COLUMNS
...
+
+#back-to-top:hover{
+  background-color: #333;
+  color: #fff;
+}
+
+#back-to-top.display{
+  opacity: 1;
+  visibility: visible;
+}
```
Comment (jsnshrmn) [inline TWLight/static/css/local.css:775]: files should end with a newline

**Comment by jsnshrmn** [comment]
Also, we've moved to an anonymizing reverse proxy for external assets, so it would be great if you updated the URLs to match what is in #447. Since you had no way of knowing that this change was about to happen, I'd be happy to make that update on your behalf if you don't want to.

File: TWLight/static/css/local.css (hunk 1 above)
Comment (lalit97) [reply TWLight/static/css/local.css]: added a newline :)

**Comment by lalit97** [comment]
> This occludes the footer navigation in mobile views. That needs to be resolved

Fixed it, now working well in mobile view.

> When I load the page without JavaScript, this feature disappears. Please use JavaScript for progressive enhancement where possible. Since this is just navigation between page anchors, it should work without JavaScript.

removed the javascript part.

**Comment by lalit97** [comment]
> Also, we've moved to an anonymizing reverse proxy for external assets, so it would be great if you updated the URLs to match what is in #447. Since you had no way of knowing that this change was about to happen, I'd be happy to make that update on your behalf if you don't want to.

I have removed the urls, since they were already present in `base.html`. Thank you for such a detailed review. I have fixed all the points you have mentioned :)

File: TWLight/resources/templates/resources/partner_filter.html [hunk 2]
```diff
@@ -49,4 +49,8 @@ <h2>{% trans "Browse Partners" %}</h2>
       </div>
     {% endfor %}
   </div>
+
+  <a id="back-to-top" href="#" class="btn btn-light btn-sm">
```
Comment (suecarmol) [inline TWLight/resources/templates/resources/partner_filter.html:53]: Can you add a tooltip that says `Back to top` along with its corresponding translator comment?

File: TWLight/static/css/local.css [hunk 3]
```diff
@@ -752,3 +753,24 @@ CLEARING COLUMNS
     clear: left;
   }
 }
+
+/* For 'back to top' button on /partners/ */
+#back-to-top {
+  display: inline-block;
+  border-radius: 4px;
+  position: fixed;
+  bottom: 210px;
```
Comment (suecarmol) [inline TWLight/static/css/local.css:762]: I'm going back and forth on whether this should be lower (around 170px). At 170px, the button doesn't cover the footer in any device I have tested it on. Is there a reason why you chose 210px?

File: TWLight/resources/templates/resources/partner_filter.html (hunk 2 above)
Comment (lalit97) [reply TWLight/resources/templates/resources/partner_filter.html]: added a tooltip :)

File: TWLight/static/css/local.css (hunk 3 above)
Comment (lalit97) [reply TWLight/static/css/local.css]: Reduced the value to 170px you are right it is working fine on this value at all devices :)

**Pull request merged**